_sass/_documents.scss

```diff
@@ -5,90 +5,97 @@ div #doccontent {
     padding-left: 30px;
     margin-right: 30px;
     height: 94vh;
-	padding-right: 10px;
-	h1 {
-		font-size: 45px;
-		font-family: $fontsMontserrat;
-		font-weight: 700;
-		line-height: normal;
-		letter-spacing: 0em;
-		padding-bottom: 12px;
-		text-align: left;
-	}
-	h2 {
-		font-size: 35px;
-		font-family: $fontsMontserrat;
-		font-weight: 700;
-		line-height: normal;
-		letter-spacing: 0em;
-		padding-bottom: 12px;
-		text-align: left;
-	}
-	h3 {
-		font-size: 28px;
-		font-family: $fontsMontserrat;
-		font-weight: 700;
-		text-align: left;
-		line-height: 40px;
-		letter-spacing: 0em;
-	}
+    padding-right: 10px;
 
-	h4 {
-		font-size: 24px;
-		font-family: $fontsMuli;
-		text-align: left;
-		line-height: 32px;
-		letter-spacing: 0em;
-	}
+    h1 {
+        font-size: 45px;
+        font-family: $fontsMontserrat;
+        font-weight: 700;
+        line-height: normal;
+        letter-spacing: 0em;
+        padding-bottom: 12px;
+        text-align: left;
+    }
 
-	h5 {
-		font-size: 20px;
-		line-height: 24px;
-	}
+    h2 {
+        font-size: 35px;
+        font-family: $fontsMontserrat;
+        font-weight: 700;
+        line-height: normal;
+        letter-spacing: 0em;
+        padding-bottom: 12px;
+        text-align: left;
+    }
 
-	li {
-		color: $gray;
-		font-family: $fontsMuli;
-		font-size: 16px;
-		line-height: 25px;
-		letter-spacing: 0em;
-	}
+    h3 {
+        font-size: 28px;
+        font-family: $fontsMontserrat;
+        font-weight: 700;
+        text-align: left;
+        line-height: 40px;
+        letter-spacing: 0em;
+    }
 
-	table {
-		color: $gray;
-		font-family: $fontsMuli;
-		font-size: 16px;
-		line-height: 25px;
-		letter-spacing: 0em;
-		thead {
-			color: black;
-		}
-	}
+    h4 {
+        font-size: 24px;
+        font-family: $fontsMuli;
+        text-align: left;
+        line-height: 32px;
+        letter-spacing: 0em;
+    }
 
-	code {
-		color: black;
-		font-family: Monospace;
-		font-size: 14px;
-	}
+    h5 {
+        font-size: 20px;
+        line-height: 24px;
+    }
 
-	ul {
-		list-style-type: disc;
-		margin-left: 1.5em;
-		padding-left: 1.5em;
-	}
+    li {
+        color: $gray;
+        font-family: $fontsMuli;
+        font-size: 16px;
+        line-height: 25px;
+        letter-spacing: 0em;
+    }
 
-	ol {
-		list-style-type: decimal;
-		margin-left: 1.5em;
-		padding-left: 1.5em;
-	}
+    table {
+        color: $gray;
+        font-family: $fontsMuli;
+        font-size: 16px;
+        line-height: 25px;
+        letter-spacing: 0em;
+        thead {
+            color: black;
+        }
+    }
 
-	p {
-		padding-bottom: 10px;
-		padding-top: 10px;
-	}
+    pre {
```

Comment:
you can see it here:

https://github.com/GoogleCloudPlatform/kubernetes/pull/11504/files?w=0